tests/integration/test_integration.py

```diff
@@ -795,19 +801,67 @@ def test_pex_resource_bundling():
             assert stdout == b"hello\n"
 
 
-def test_entry_point_verification_3rdparty(tmpdir):
-    # type: (Any) -> None
+@pytest.mark.parametrize(
+    "layout", [pytest.param(layout, id=layout.value) for layout in Layout.values()]
```

Comment:
Totally minor, but are you aware that `pytest.mark.parametrize` supports an `ids` parameter that can take a function, so these could potentially be something like:

```python
@pytest.mark.parametrize("layout", Layout.values(), ids=lambda layout: layout.value)
...
```

Reply:
I was not aware. I'm not sure I like that though! It's not obvious the two align when you split things up like that. 10 values, 6 ids. Clearly they align in your example, but it takes a second to make sure of that.

Reply:
Oh, I read poorly. It's 1 function mapped over all the values. Gotcha.

Reply:
Yeah, in addition to the function, it also supports passing a list with exactly the right number of IDs, but I don't like that possibility either.